README.md

```diff
@@ -187,6 +187,23 @@ spec:
   imagePullPolicy: Always
 ```
 
+## Self Signed Certificates Configuration
+
+Administrator can specify [webhook self signed certificates configuration](https://pkg.go.dev/github.com/qinqon/kube-admission-webhook@v0.12.0/pkg/certificate?tab=doc#Options)
```

Comment:
I think we should mention which components are currently using this certification manager service